frontend/src/views/ManagementPage/index.vue

```diff
@@ -11,6 +11,15 @@
           Modifier mon profil
         </v-btn>
       </p>
+
+      <div v-if="loggedUser.isElected">
+        Vous avez un compte élu / élue, vous pouvez voir
+        <router-link text color="primary" :to="{ name: 'ElectedCanteens' }">
+          les cantines de votre territoire
+        </router-link>
+        .
```

Comment:
It would be great to figure out a way to avoid the space between the link and the full stop across the board. Maybe for another PR